composer.json

```diff
@@ -23,14 +23,14 @@
         }
     ],
     "require":     {
-        "php": "^5.4|^7",
+        "php": ">=5.5.9",
         "psr/cache": "~1.0",
-        "cache/tag-interop": "^1.0"
+        "cache/tag-interop": "^1.0",
+        "symfony/phpunit-bridge": "^4.4"
```

Comment:
`4.4 || ^5.0`?

Reply:
Will update as soon as there's agreement on the PHP version.

Reply:
Symfony 5's phpunit bridge has the same requirements as 4's, and 4.4 and 5.0 have the same features, ignoring deprecated ones.